requirements.txt

```diff
@@ -1,2 +1,19 @@
-elasticsearch-curator==5.4.0
-easydict
+boto3==1.14.17
```

Comment:
It looks like a lot of unnecessary dependencies got pulled in, can you prune this down to just the dependencies this module relies on?

Specifically, don't pin any transitive dependencies, let the modules we depend on pull those in for us, only focus on direct dependencies.

Reply:
Done in  53b3c12 but I would've liked fixing transitive dependencies to have repeatable installs.